src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -463,7 +463,7 @@ def export(
                 model_cfg = model._config.get("model", model._config)  # noqa: SLF001
             else:
                 raise NotImplementedError
-        elif self.dumped_config.get("model", None) and self.dumped_config["model"] is not None:
+        elif self.dumped_config.get("model", None) is not None:
```

Comment:
Nitpick: might be more simple.
```suggestion
        elif "model" in self.dumped_config:
```